src/gordon_gcp/plugins/enricher.py

```diff
@@ -156,16 +157,14 @@ def __init__(self, config, http_client, success_channel, error_channel):
                f'{exception.__class__.__name__}: {exception}')
         raise exceptions.GCPGordonError(msg)
 
-    async def _create_A_rrecord(self, event_data, instance_data, msg_logger):
-        dns_zone = self.config['dns_zone']
-        hostname = event_data['resourceName'].split('/')[-1]
+    async def _create_A_rrecord(self, fqdn, event_data, instance_data,
+                                msg_logger):
```

Comment:
I guess event data isn't needed anymore?

Reply:
This didn't change with this PR, but I also don't understand why this method is async

Reply:
Well, it might make sense to make all of the record creators async in case they need to use aiohttp or something.  Can't make only one async.